alacritty/src/main.rs

```diff
@@ -88,7 +89,10 @@ fn main() {
     // Load configuration file
     let config_path = options.config_path().or_else(config::installed_config);
     let config = config_path.map(config::load_from).unwrap_or_else(Config::default);
-    let config = options.into_config(config);
+    let config = options.into_config(config).unwrap_or_else(|err| {
+        error!(target: LOG_TARGET_CONFIG, "Unable to apply command line parameters {}", err);
+        Config::default()
+    });
```

Comment:
Resetting to the default config, only because the path isn't valid seems unnecessarily hostile. Why would we throw away all that good info, just because our working directory isn't good?